tooling/nargo_cli/build.rs

```diff
@@ -281,10 +288,16 @@ fn generate_compile_success_empty_tests(test_file: &mut File, test_data_dir: &Pa
         };
         let test_dir = &test_dir.path();
 
+        let non_elaborator_ignored = if IGNORED_NON_ELABORATOR_TESTS.contains(&test_name.as_str()) {
```

Comment:
I found `contains` to be the best option here and this is only temporary until we move to the elaborator fully. Did you have something else in mind?

Reply:
I think @michaeljklein was thinking of string contains rather than array contains which is used here. We're still checking the full test name in the array is `test_name`.